roman-amy/server.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+const express = require('express');
+const bodyParser = require('body-parser');
+const app = express();
+
+const PORT = process.env.PORT || 3000;
+//Comment: explain why our files are in a "public" directory now and how ExpressJS serves our local files.
+//all of our front-end files are in the public folder to keep it separate from the server-side which normally would happen when using a database. You cannot run two servers from the same directory.
```

Comment:
It's possible to run two servers from the same directory--that isn't the issue here, and we're only running one server. Instead, we want to make sure we only serve the files that belong on the frontend, so we isolate those files into the public directory and use the below line to serve them.